source/server-selection/server-selection.rst

```diff
@@ -521,6 +526,15 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
+`````
+
+The read preference ``hedge`` parameter is an object that configures how the
+server will perform hedged reads. It consists of the following keys:
+
+- ``enabled`` (default true): This enables hedging
```

Comment:
I think the earlier terminology about `hedge` defaulting to "empty" might lead one to believe that it's enabled by default if this is also defaulting to true.

These are the cases I'm wondering about:

 * Read preference omits `hedge` option. I think it's obvious that hedged reads won't be used at all.
 * Read preference specifies empty document for `hedge` option. This section suggests that `enabled` and `delay` will both default to true. This definitely seems like an edge case, but something to clarify in the spec.
 * Read preference specifies `hedge` document with at least one option (`enabled` OR `delay` in the current version). In that case, the other option would default accordingly.

Regarding defaults, I think we should also specify whether drivers should be filling in these defaults if ever a document is provided for `hedge` or if this is just reporting what the current server-side defaults are. I think it's more intuitive for drivers not to fill in defaults on their own. That agrees with existing conventions in the [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst) where we say "the default is to not send a value".

Reply:
Yes, the wording needs changing. I think we need to clarify omitting the `hedge` option entirely and submitting an empty document, in which case hedging is enabled with default values. Consider this paragraph:
> To explicitly enable hedging, an empty document must be sent to the server for the `hedge` option. This enables hedging with the default values specified above. To not enable hedging, the `hedge` option must be omitted entirely.

Regarding default values I would add the following sentence:
> Drivers MUST NOT fill in default values for any keys in the ``hedge`` document if they are omitted.